src/chain/store/index.rs

```diff
@@ -205,6 +205,7 @@ mod tests {
     use crate::db::MemoryDB;
     use crate::utils::db::CborStoreExt;
 
+    // TODO(aatifsyed): refactor this
```

Comment:
Is it a TODO for this PR or for later? If it's the latter, let's have a tracking issue.